House-of-glass/src/components/Cards/WishListCard/WishListCard.jsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react'
+import Ratings from '../../Ratings/Ratings'
+import { useAuth } from 'Utils/AuthContext';
+
+
+
+export default function WishListCard({Item}) {
+  const {name,price,discount,fastDelivery,imageUrl,rating} = Item;
+  const {userDetails,setUserDetails}= useAuth();
+  const {cartList} = userDetails;
+
+
+  const addQty = (Item) =>{
+    cartList.find((cartItem)=>cartItem._id===Item._id)?setUserDetails({...userDetails, cartList:cartList.map((cartItem)=>(cartItem._id===Item._id)?{...cartItem,count: cartItem.count+1}:cartItem)}):
+    setUserDetails({...userDetails, cartList:[...cartList,{...Item, count:1}]});
+  }
+
+  const decreaseQty = (Item) =>{
+    cartList.find((cartItem)=>cartItem._id===Item._id)?setUserDetails({...userDetails, cartList:cartList.map((cartItem)=>(cartItem._id===Item._id)?{...cartItem,count: cartItem.count-1}:cartItem)}):
+    setUserDetails({...userDetails, cartList:[...cartList,{...Item, count:1}]});
+  }
```

Comment:
The format seems little hard to determine as to what is happening here right now so do format it later on. But why are we using these functions when the changes are happening through backend functions which you mentioned above

Reply:
Thanks for pointing this out. This was done as a test to see how the backend function might be performing. 
Will be replacing it with the backend defined functional call.